### DISCUSSION TITLE
Cross Compiling with eMMC

### DISCUSSION THREAD

**Discussion by doublejz** [geerlingguy/raspberry-pi-pcie-devices#183] [Q&A]
I was following this article to cross compile raspberry pi for additional drivers but go stuck.

https://github.com/geerlingguy/raspberry-pi-pcie-devices/tree/master/extras/cross-compile/legacy-vagrant

I see that the instructions say to remove the SD card from the CM4 and mount it on another device to load the new kernel files and DTBs. My CM4 has eMMC on it and I haven't been able to ever disable eMMC boot and mount the eMMC as an accessible storage device. Actually it never even shows up in fdisk even though I see it being detected in dmesg. Is there a trick or way to be able to mount an eMMC to push the new files over?

**Comment by geerlingguy** [comment]
See [How to flash Raspberry Pi OS onto the Compute Module 4 eMMC with usbboot](https://www.jeffgeerling.com/blog/2020/how-flash-raspberry-pi-os-compute-module-4-emmc-usbboot). You'll need to use Raspberry Pi's `usbboot` utility to mount the eMMC drive on your computer as a disk, then you can manage files on it.
